lite/core/mir/fusion/__xpu__logit_fuse_pass.cc

```diff
@@ -0,0 +1,119 @@
+
+class XPULogitFuser : public FuseBase {
+ public:
+  void BuildPattern() override {
+    auto* input = VarNode("input")->assert_is_op_input("clip", "X")->AsInput();
+    auto* clip1 = OpNode("clip1", "clip")->AsIntermediate();
+    auto* clip1_out = VarNode("clip1_out")
+                          ->assert_is_op_output("clip", "Out")
+                          ->assert_is_op_input("fill_any_like", "X")
+                          ->assert_is_op_input("elementwise_div", "Y")
+                          ->AsIntermediate();
+    auto* fill_any_like =
+        OpNode("fill_any_like", "fill_any_like")->AsIntermediate();
+    auto* fill_any_like_out = VarNode("fill_any_like_out")
+                                  ->assert_is_op_output("fill_any_like", "Out")
+                                  ->assert_is_op_input("elementwise_div", "X")
+                                  ->AsIntermediate();
+    auto* elementwise_div = OpNode("elementwise_div", "elementwise_div")
+                                ->assert_op_attr<int>("axis", -1)
+                                ->AsIntermediate();
+    auto* elementwise_div_out =
+        VarNode("elementwise_div_out")
+            ->assert_is_op_output("elementwise_div", "Out")
+            ->assert_is_op_input("scale", "X")
+            ->AsIntermediate();
+    auto* scale1 = OpNode("scale1", "scale")
+                       ->assert_op_attr<float>("scale", 1.)
+                       ->AsIntermediate();
+    auto* scale1_out = VarNode("scale1_out")
+                           ->assert_is_op_output("scale", "Out")
+                           ->assert_is_op_input("clip", "X")
+                           ->AsIntermediate();
+    auto* clip2 = OpNode("clip2", "clip")->AsIntermediate();
+    auto* clip2_out = VarNode("clip2_out")
+                          ->assert_is_op_output("clip", "Out")
+                          ->assert_is_op_input("log", "X")
+                          ->AsIntermediate();
+    auto* log = OpNode("log", "log")->AsIntermediate();
+    auto* log_out = VarNode("log_out")
+                        ->assert_is_op_output("log", "Out")
+                        ->assert_is_op_input("scale", "X")
+                        ->AsIntermediate();
+    auto* scale2 = OpNode("scale2", "scale")
+                       ->assert_op_attr<float>("scale", -1.)
+                       ->AsIntermediate();
+
+    auto* out = VarNode("out")->assert_is_op_output("scale", "Out")->AsOutput();
+
+    *input >> *clip1 >> *clip1_out;
+    *clip1_out >> *fill_any_like >> *fill_any_like_out >> *elementwise_div;
+    *clip1_out >> *elementwise_div;
+    *elementwise_div >> *elementwise_div_out >> *scale1 >> *scale1_out;
+    *scale1_out >> *clip2 >> *clip2_out >> *log >> *log_out >> *scale2 >> *out;
+  }
+
+  void InsertNewNode(SSAGraph* graph, const key2nodes_t& matched) override {
+    cpp::OpDesc op_desc = *matched.at("clip1")->stmt()->op_info();
+    op_desc.mutable_inputs()->clear();
+    op_desc.mutable_outputs()->clear();
```

Comment:
```suggestion
    cpp::OpDesc op_desc;
```
这里没必要复制clip1的OpDesc，新建就可以，也避免做clear的工作

Reply:
done